### ISSUE TITLE
The loss changes largely and unstablely

### ISSUE THREAD

**Issue by skyloveblue** [balancap/SSD-Tensorflow#169]
The loss changes largely and unstablely when I train SSD from vgg using voc2007+2012, how should I tune hyper-parameters?

**Comment by moonyoung430** [comment]
I have the same problem,do you solve it?

**Comment by zzks** [comment]
me too, any updates? my new data set also cannot work.
